### PR TITLE
Add a proper README.md

### LINKED ISSUE #8: Add proper README and other github stuff
**Current behavior:** 
Description, tags and README are empty.

**Expected behavior:**  
- Description should briefly introduce the extension
- Tags should be filled with relevant tags for search purposes
- README should introduce the project and contributing process / usage process

**Actions pending:**
- [x] Write a description
- [x] Write tags
- [ ] Write a README

### PR DESCRIPTION
**Pull request by MatonAnthony**
This PR should closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 # doc-linter-webextension
 
-[![Greenkeeper badge](https://badges.greenkeeper.io/mdn/doc-linter-webextension.svg)](https://greenkeeper.io/)
\ No newline at end of file
+[![Greenkeeper badge](https://badges.greenkeeper.io/mdn/doc-linter-webextension.svg)](https://greenkeeper.io/) [![Build Status](https://travis-ci.org/mdn/doc-linter-webextension.svg?branch=master)](https://travis-ci.org/mdn/doc-linter-webextension)
+
+This WebExtension is used to lint MDN Document.
```
Comment (Elchi3) [inline README.md]: MDN documents.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+The rules used by the extension to lint the documents are available as part of a separate package (https://github.com/mdn/doc-linter-rules)
+
+## Compatibility
+This WebExtension is compatible with :
```
Comment (Elchi3) [inline README.md]: No space between with and the colon. "with:"
Same below.

**Comment by Elchi3** [comment]
Also, add a screenshot like the old add-on did? :)

**Comment by MatonAnthony** [comment]
I don't understand why there is one Travis failing and the other passing.
The failing one is complaining on files I didn't even touched ...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+We are happy to accept new contributions to this webextension.
+
+## Types of contributions
+
+There is many ways to contribute to this repository, for example:
```
Comment (Elchi3) [inline CONTRIBUTING.md]: There are

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  - Adding new features
+  - Opening issues about existing problems
+  - Fixing issues
+  - Reviewing pull requests
+
+## Development guidelines
+
+We have a few development guidelines regarding the code, that we try to follow as much as possible.
+  - We try to stick as much as possible with ES6 (https://kangax.github.io/compat-table/es6/) and ES2016+ (http://kangax.github.io/compat-table/es2016plus/)
```
Comment (Elchi3) [inline CONTRIBUTING.md]: Full stop missing "(http://kangax.github.io/compat-table/es2016plus/)."

File: README.md [hunk 5]
```diff
@@ -1,3 +1,23 @@
...
+
+This WebExtension is used to lint MDN Documents.
+The rules used by the extension to lint the documents are available as part of a separate package (https://github.com/mdn/doc-linter-rules)
+
+![Extension sidebar caption](https://github.com/mdn/doc-linter-webextension/blob/master/sidebar.png)
+
+## Compatibility
+This WebExtension is compatible with:
+  - Firefox 54+
+This WebExtension might be compatible with :
```
Comment (Elchi3) [inline README.md]: no space between with and colon "with:"

File: README.md [hunk 6]
```diff
@@ -1,3 +1,23 @@
...
+  - Opera 46
+This WebExtension is not compatible at all with :
```
Comment (Elchi3) [inline README.md]: same here

File: README.md [hunk 7]
```diff
@@ -1,3 +1,23 @@
...
+  - Google Chrome (Sidebar API is not implemented)
+  - Microsoft Edge (Sidebar API is not implemented)
+  - Safari
+
+## Contributing
+
+We are happy to accept contributions to the WebExtension. If you have any questions, we will be glad to answer them on irc.mozilla.org #mdn.
+See also the [Contribution guidelines](https://github.com/mdn/doc-linter-webextension/blob/master/CONTRIBUTING.md)
```
Comment (Elchi3) [inline README.md]: Full stop in the end.

**Pull request merged**